rastervision/backend/pytorch_chip_classification.py

```diff
@@ -217,17 +218,59 @@ def train(self, tmp_dir):
         num_labels = len(databunch.label_names)
         model = get_model(
             self.train_opts.model_arch, num_labels, pretrained=True)
+        if torch.cuda.device_count() > 1:
+            model = nn.DataParallel(model)
```

Comment:
I think we should add a `multi_gpu` option, and it's default value should be True, but it seems like it would be good to give people the option to turn this off.

Reply:
Would you agree that the best place to have this flag would be in the backend configuration? i.e. `rv.BackendConfig.builder(rv.PYTORCH_CHIP_CLASSIFICATION).with_multi_gpu(True)` would enable multi GPU? In that case I would pass the flag on in:

- `PyTorchChipClassificationConfigBuilder` from `pytorch_chip_classification_config.py` via a `with_multi_gpu()` method, correct?

Reply:
I would rather add an option to `with_train_options` here https://github.com/azavea/raster-vision/blob/master/rastervision/backend/pytorch_chip_classification_config.py#L53